pages/linux/gopass.md

```diff
@@ -0,0 +1,36 @@
+# gopass
+
+> Standard Unix Password Manager for Teams. Written in Go.
+> Homepage: <https://www.gopass.pw>.
+
+- Initialise the configuration settings:
+
+`gopass init`
+
+- Show all stores:
+
+`gopass mounts`
+
+- Mount a shared git store:
+
+`gopass mounts add {{store_name}} {{git_repo_url}}`
+
+- Search interactively using a keyword:
+
+`gopass show {{keyword}}`
+
+- Search using a keyword:
+
+`gopass find {{keyword}}`
+
+- Sync all mounted stores:
+
+`gopass sync`
+
+- Show a particular password entry:
+
+`gopass {{store_name}}/{{path/to/directory}}/{{email@email.com}}`
+
+- Create a new entry:
```

Comment:
I'd suggest moving this to the top as the second example since it looks more common/basic than others. Having it as the last example is counter-intuitive. Do you agree?